internal_testsuites/golang/testsuite/stream_logs_test/stream_logs_test.go

```diff
@@ -125,7 +121,9 @@ func TestStreamLogs(t *testing.T) {
 
 		require.NoError(t, testEvaluationErr)
 		for userServiceUuid := range requestedServiceUuids {
-			require.Equal(t, expectedLogLines, receivedLogLinesByService[userServiceUuid])
+			for logNum, expectedLogLine := range expectedLogLinesByService[userServiceUuid] {
```

Comment:
weirdly - k3s clusters (what our CI tests run on) use a different log format than minikube, it adds a "<timestamp> stdou F" before the log message. First I tried to change the k3s cluster to use the json logging format like kubernetes (matches docker - only log not other data is added to log line) but the setting wasn't working so instead I changed the test to ensure the log is at least contained within logs picked up.